### DISCUSSION TITLE
Εργαστήριο Παρασκευής 18-02-2022

### DISCUSSION THREAD

**Discussion by diogenisAl** [courses-ionio/sw#1035] [Announcements]
Καλησπέρα & καλό εξάμηνο! :wave:

Λόγω του μεγάλου πλήθους των φοιτητών/-ιών και του περιορισμένου χώρου, για λόγους ασφάλειας το αυριανό δίωρο εργαστήριο θα χωριστεί σε δύο ίδια μονόωρα:
* την πρώτη ώρα παρακαλούνται να προσέλθουν **μόνο** όσοι/-ες έχουν Αριθμό Μητρώου (**ΑΜ**) που λήγει σε **μονό** αριθμό,
* τη δεύτερη ώρα παρακαλούνται να προσέλθουν **μόνο** όσοι/-ες έχουν Αριθμό Μητρώου (**ΑΜ**) που λήγει σε **ζυγό** αριθμό.

Καλή αρχή!

**Comment by p14pall** [comment]
Καλησπέρα σας, συγνώμη που γράφω εδώ, απλώς είναι κλειστά τα υπόλοιπα.
Γνωρίζετε πότε θα ανακοινωθούν οι βαθμολογίες της επαναληπτικής εξέτασης του sw?
Αν δεν κάνω λάθος εσείς κάνατε merge το PR μου.
Επείγει σχετικά γιατί παίρνω πτυχίο με την ολοκλήρωση του μαθήματος.
Ευχαριστώ προκαταβολικά

**Reply by diogenisAl** [reply]
Καλησπέρα @p14pall 
Δεν γνωρίζω κάτι για τις βαθμολογίες. Θα πρέπει να επικοινωνήσεις με τον @epidrome
